website/blog/2024-05-22-latest-dbt-stability-improvement-innovation.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Latest dbt: a stable foundation for improvement & innovation"
+description: "Over the past 6 months, we've laid a stable foundation for continuously improving dbt."
+slug: latest-dbt-stability
+
+authors: [michelle_ark, chenyu_li, colin_rogers]
```

Comment:
@jtcohen6 shouldn't you be an author here as well? 😄

Reply:
azzam & i are very happy ghosts ;)